packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -545,69 +541,56 @@ def test_app_networking_mode_with_defined_container_port(dcos_api_session, netwo
         assert 'pong' in r.json()
 
 
-@retrying.retry(wait_fixed=2000,
-                stop_max_delay=100 * 2000,
-                retry_on_exception=lambda x: True)
-def geturl(url):
-    rs = requests.get(url)
-    assert rs.status_code == 200
-    r = rs.json()
-    log.info('geturl {} -> {}'.format(url, r))
-    return r
-
-
 def test_l4lb(dcos_api_session):
     '''Test l4lb is load balancing between all the backends
        * create 5 apps using the same VIP
-       * get uuid from the VIP in parallel from many threads
```

Comment:
Threads don't run in parallel due to GIL 😕

Reply:
They don't run physically simultaneously, yes.

They run concurrently, though :-)